src/App.js

```diff
@@ -1,25 +1,18 @@
-import logo from "./logo.svg";
-import "./App.css";
+/* eslint-disable import/extensions */
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Book from "./components/Book";
```

Comment:
- [ ] When the application is run in the local environment, it complains about this import as seen below. Please kindly fix this bug and make sure that the application works as intended.

![image](https://user-images.githubusercontent.com/4247740/157560598-f25d6046-2699-4771-b656-1ab7b5fd8831.png)